supabase/migrations/20230124013902_add_participants.sql

```diff
@@ -0,0 +1,25 @@
+create table "public"."participants" (
+    "id" uuid not null,
+    "created_at" timestamp with time zone default now(),
+    "first_name" character varying,
+    "last_name" character varying,
+    "DOB" character varying,
+    "school" character varying,
+    "major" character varying,
+    "graduation_year" character varying,
+    "resume" character varying,
```

Comment:
better way to ensure tight FK integrity is to link this to the `storage` table native to Supabase which keeps track of objects in Storage in tables
![image](https://user-images.githubusercontent.com/64177421/214476695-5e394ef0-6005-4a13-bdcb-340b549870f5.png)